meson_options.txt

```diff
@@ -215,6 +215,8 @@ option('pamlibdir', type : 'string',
        description : 'directory for PAM modules')
 option('pamconfdir', type : 'string',
        description : 'directory for PAM configuration ["no" disables]')
+option('ssh-generator', type : 'boolean',
```

Comment:
This seems overkill to me - it doesn't add dependencies or anything, and it can simply be skipped when packaging